gulpfile.js/tasks/build.js

```diff
@@ -0,0 +1,58 @@
+const { src, dest, parallel } = require('gulp');
+
+const sass = require('gulp-sass');
+const rename = require('gulp-rename');
+const postcss = require('gulp-postcss');
+const cssnano = require('cssnano');
+
+const rollup = require('rollup');
+const path = require('path');
+const multiEntry = require('@rollup/plugin-multi-entry');
+const nodeResolve = require('@rollup/plugin-node-resolve');
+const { terser } = require('rollup-plugin-terser');
+
+sass.compiler = require('sass');
+
+const outDir = 'dist';
+
+const buildSass = () => {
+  return src('./src/index.scss')
+    .pipe(
+      sass({
+        // Import Theo design tokens as SCSS variables
+        importer: require('../../.storybook/theo-importer'),
```

Comment:
Since this is now a dependency of both the build task and Storybook, should we move it out of the `.storybook` directory?

Reply:
Fixed in https://github.com/cloudfour/cloudfour.com-patterns/commit/78210ab34358d7f7e002086460254cb99839ed51